rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    person.add_rental(self)
+    book.add_rental(self)
+  end
+end
```

Comment:
Whenever I try to create a rental Im encountering an error. It would be nice if you fixed this to enable creation of a rental. 

Here is the error:
![Screenshot 2023-11-25 at 6 13 19 PM](https://github.com/BHObserver/OOP-School-Library/assets/79506105/1052538f-8c04-4d8c-be20-7ef68c9ec287)